openmdao/surrogate_models/kriging.py

```diff
@@ -46,17 +50,24 @@ class KrigingSurrogate(SurrogateModel):
         Standard deviation of training model response values, normalized.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, lapack_driver='gesvd', **kwargs):
         """
         Initialize all attributes.
 
         Parameters
         ----------
+        lapack_driver : str, optional
```

Comment:
KrigingSurrogate has an OptionsDictionary, so this should be declared as an option in `_declare_options` instead of handling it as a constructor argument.

Reply:
Good point! Fixing